src/main/java/abr/user_login_abr/UserLogResponseModel.java

```diff
@@ -13,7 +14,8 @@ public class UserLogResponseModel {
     String passWord;
```

Comment:
Inconsistent naming of password, I see a few references with passWord and others as password.